packages/table/test/formatsTests.tsx

```diff
@@ -115,7 +119,11 @@ describe("Formats", () => {
                 The fair Ophelia! -- Nymph, in thy orisons
                 Be all my sins remembered.
             `;
-            const comp = harness.mount(<TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>);
+            const comp = harness.mount(
+                <div className={Classes.TABLE_NO_WRAP_TEXT}>
+                    <TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>
+                </div>,
```

Comment:
I'm actually totally okay with this. The idea is that truncatedFormat can live in both a wrapped and unwrapped div (like a `<Cell>`. So, this test actually confirms that.